apps/server/src/infra/etherpad-client/etherpad-client.adapter.spec.ts

```diff
@@ -995,11 +995,34 @@ describe(EtherpadClientAdapter.name, () => {
 			it('should throw EtherpadErrorLoggableException', async () => {
 				const groupId = setup();
 
-				const exception = new EtherpadErrorLoggableException(EtherpadErrorType.BAD_REQUEST, { padId: groupId }, {});
+				const exception = new EtherpadErrorLoggableException(EtherpadErrorType.INTERNAL_ERROR, { padId: groupId }, {});
 				await expect(service.deleteGroup(groupId)).rejects.toThrowError(exception);
 			});
 		});
 
+		describe('when deleteGroupUsingPOST returns invalid etherpad error', () => {
+			const setup = () => {
+				const groupId = 'groupId';
+				const response = createMock<AxiosResponse<InlineResponse2001>>({
+					data: {
+						code: EtherpadResponseCode.BAD_REQUEST,
+						message: 'sessionID does not exist',
+						data: {},
+					},
+				});
+
+				groupApi.deleteGroupUsingPOST.mockResolvedValue(response);
+
+				return groupId;
+			};
+
+			it('should return successfull', async () => {
+				const groupId = setup();
+
+				await service.deleteGroup(groupId);
+			});
```

Comment:
I think a test, that the function call to `deleteGroup` returned successfull is missing.